>From c417f21e5a720f8bfe9ee222948ae8c59d4a485b Mon Sep 17 00:00:00 2001
From: Adrian Reber <adrian.reber@hs-esslingen.de>
Date: Wed, 20 Nov 2013 14:50:12 +0100
Subject: [PATCH] Trying to get the C/R code to compile again. (headers)

In a first step to enable C/R in Open MPI again the goal was
to get it to compile. This commit only fixes the #include directives.
---
 orte/mca/ess/base/ess_base_std_app.c           | 1 +
 orte/mca/ess/base/ess_base_std_orted.c         | 1 +
 orte/mca/ess/env/ess_env_module.c              | 2 +-
 orte/mca/ess/hnp/ess_hnp_module.c              | 1 +
 orte/mca/rml/oob/rml_oob_component.c           | 3 +++
 orte/mca/snapc/full/snapc_full_global.c        | 2 +-
 orte/mca/snapc/full/snapc_full_local.c         | 2 +-
 orte/mca/sstore/stage/sstore_stage_app.c       | 1 +
 orte/mca/sstore/stage/sstore_stage_component.c | 1 +
 9 files changed, 11 insertions(+), 3 deletions(-)

diff --git a/orte/mca/ess/base/ess_base_std_app.c b/orte/mca/ess/base/ess_base_std_app.c
index c670aaa..dbbb2f4 100644
--- a/orte/mca/ess/base/ess_base_std_app.c
+++ b/orte/mca/ess/base/ess_base_std_app.c
@@ -53,6 +53,7 @@
 #include "orte/mca/errmgr/base/base.h"
 #if OPAL_ENABLE_FT_CR == 1
 #include "orte/mca/snapc/base/base.h"
+#include "orte/mca/sstore/base/base.h"
 #endif
 #include "orte/mca/state/base/base.h"
 #include "orte/util/proc_info.h"
diff --git a/orte/mca/ess/base/ess_base_std_orted.c b/orte/mca/ess/base/ess_base_std_orted.c
index 3f228fc..ce11f33 100644
--- a/orte/mca/ess/base/ess_base_std_orted.c
+++ b/orte/mca/ess/base/ess_base_std_orted.c
@@ -56,6 +56,7 @@
 #include "orte/mca/errmgr/errmgr.h"
 #if OPAL_ENABLE_FT_CR == 1
 #include "orte/mca/snapc/base/base.h"
+#include "orte/mca/sstore/base/base.h"
 #endif
 #include "orte/mca/filem/base/base.h"
 #include "orte/util/proc_info.h"
diff --git a/orte/mca/ess/env/ess_env_module.c b/orte/mca/ess/env/ess_env_module.c
index c4ce756..6a71230 100644
--- a/orte/mca/ess/env/ess_env_module.c
+++ b/orte/mca/ess/env/ess_env_module.c
@@ -57,7 +57,7 @@
 #include "orte/mca/rmaps/base/base.h"
 #if OPAL_ENABLE_FT_CR == 1
 #include "orte/mca/snapc/base/base.h"
-#include "orte/mca/db/db.h"
+#include "opal/mca/db/db.h"
 #endif
 #include "orte/mca/filem/base/base.h"
 #include "orte/util/proc_info.h"
diff --git a/orte/mca/ess/hnp/ess_hnp_module.c b/orte/mca/ess/hnp/ess_hnp_module.c
index 4a7ccb7..a6f1777 100644
--- a/orte/mca/ess/hnp/ess_hnp_module.c
+++ b/orte/mca/ess/hnp/ess_hnp_module.c
@@ -69,6 +69,7 @@
 #include "orte/mca/rmaps/base/base.h"
 #if OPAL_ENABLE_FT_CR == 1
 #include "orte/mca/snapc/base/base.h"
+#include "orte/mca/sstore/base/base.h"
 #endif
 #include "orte/mca/filem/base/base.h"
 #include "orte/mca/state/base/base.h"
diff --git a/orte/mca/rml/oob/rml_oob_component.c b/orte/mca/rml/oob/rml_oob_component.c
index 4e85109..dd539cd 100644
--- a/orte/mca/rml/oob/rml_oob_component.c
+++ b/orte/mca/rml/oob/rml_oob_component.c
@@ -39,6 +39,9 @@
 #include "opal/mca/backtrace/backtrace.h"
 #include "opal/mca/event/event.h"
 
+#if OPAL_ENABLE_FT_CR == 1
+#include "orte/mca/rml/rml.h"
+#endif
 #include "orte/mca/rml/base/base.h"
 #include "orte/mca/rml/rml_types.h"
 #include "orte/mca/routed/routed.h"
diff --git a/orte/mca/snapc/full/snapc_full_global.c b/orte/mca/snapc/full/snapc_full_global.c
index 1992966..8f1317b 100644
--- a/orte/mca/snapc/full/snapc_full_global.c
+++ b/orte/mca/snapc/full/snapc_full_global.c
@@ -34,6 +34,7 @@
 #include "opal/mca/base/base.h"
 #include "opal/mca/crs/crs.h"
 #include "opal/mca/crs/base/base.h"
+#include "opal/mca/db/db.h"
 
 #include "orte/util/name_fns.h"
 #include "orte/util/proc_info.h"
@@ -45,7 +46,6 @@
 #include "orte/mca/rmaps/rmaps_types.h"
 #include "orte/mca/plm/plm.h"
 #include "orte/mca/grpcomm/grpcomm.h"
-#include "orte/mca/db/db.h"
 #include "orte/runtime/orte_wait.h"
 #include "orte/mca/errmgr/errmgr.h"
 #include "orte/mca/errmgr/base/base.h"
diff --git a/orte/mca/snapc/full/snapc_full_local.c b/orte/mca/snapc/full/snapc_full_local.c
index 895bbf6..0975d77 100644
--- a/orte/mca/snapc/full/snapc_full_local.c
+++ b/orte/mca/snapc/full/snapc_full_local.c
@@ -51,6 +51,7 @@
 #include "opal/mca/base/base.h"
 #include "opal/mca/crs/crs.h"
 #include "opal/mca/crs/base/base.h"
+#include "opal/mca/db/db.h"
 
 #include "orte/util/show_help.h"
 #include "orte/util/name_fns.h"
@@ -63,7 +64,6 @@
 #include "orte/mca/errmgr/errmgr.h"
 #include "orte/mca/routed/routed.h"
 #include "orte/mca/grpcomm/grpcomm.h"
-#include "orte/mca/db/db.h"
 
 #include "orte/mca/snapc/snapc.h"
 #include "orte/mca/snapc/base/base.h"
diff --git a/orte/mca/sstore/stage/sstore_stage_app.c b/orte/mca/sstore/stage/sstore_stage_app.c
index 50ecf48..10d38ad 100644
--- a/orte/mca/sstore/stage/sstore_stage_app.c
+++ b/orte/mca/sstore/stage/sstore_stage_app.c
@@ -47,6 +47,7 @@
 #include "orte/runtime/orte_globals.h"
 #include "orte/runtime/orte_wait.h"
 #include "orte/mca/errmgr/errmgr.h"
+#include "orte/mca/rml/rml.h"
 #include "orte/mca/rml/rml_types.h"
 
 #include "orte/mca/sstore/sstore.h"
diff --git a/orte/mca/sstore/stage/sstore_stage_component.c b/orte/mca/sstore/stage/sstore_stage_component.c
index 4dd0210..19d7c75 100644
--- a/orte/mca/sstore/stage/sstore_stage_component.c
+++ b/orte/mca/sstore/stage/sstore_stage_component.c
@@ -10,6 +10,7 @@
 
 #include "orte_config.h"
 #include "opal/util/output.h"
+#include "opal/util/opal_environ.h"
 #include "orte/constants.h"
 
 #include "orte/mca/sstore/sstore.h"
-- 
1.8.3.1


